### PR TITLE
minor copy edits

### PR DESCRIPTION
**Pull request by jkonrath-postman**
Minor stuff from my first read-through

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 # Postman Learning Center
 
-Postman is the collaboration platform for API development. And the [Postman Learning Center](https://learning.postman.com/) has the resources you need to ramp up quickly and extend your skills with Postman. Postman allows you to write API test cases in any format of your choice (JSON/XML/many more) and run them to verify and validate your API. Moreover, Postman provides collections which can improve a developer's workflow. Postman is available for several platforms including windows and Mac OS X. For more information visit [the Postman website](https://www.postman.com/).
+Postman is the collaboration platform for API development. And the [Postman Learning Center](https://learning.postman.com/) has the resources you need to ramp up quickly and extend your skills with Postman. Postman allows you to write API test cases in any format of your choice (JSON/XML/many more) and run them to verify and validate your API. Moreover, Postman provides Collections which can improve a developer's workflow. Postman is available for several platforms including Windows and Mac OS X. For more information visit [the Postman website](https://www.postman.com/).
```
Comment (claireriley) [inline README.md:5]: We typically capitalize base on [branded terms](https://postmanlabs.atlassian.net/l/c/2Wa0vYWC), so "Collections" would be lowercase unless right after "Postman". Perhaps we could link the word to https://learning.postman.com/docs/getting-started/creating-the-first-collection/ to be clearer?

Comment (jkonrath-postman) [reply README.md:5]: That's a good idea - if you want to do that. If not, no big deal.

Comment (jkonrath-postman) [reply README.md:5]: Another minor thing I just saw - we should be calling it macOS to follow Apple's trademark.

**Review by claireriley — APPROVED** [review]
Looks good to go! 🎉

**Pull request merged**